pkg/kyma/cmd/install.go

```diff
@@ -65,14 +67,21 @@ The command will:
 	cmd.Flags().StringVarP(&o.Domain, "domain", "d", "kyma.local", "domain to use for installation")
 	cmd.Flags().BoolVarP(&o.Local, "local", "l", false, "Install from sources")
 	cmd.Flags().StringVarP(&o.LocalSrcPath, "src-path", "", "", "Path to local sources to use")
+	cmd.Flags().StringVarP(&o.LocalInstallerVersion, "installer-version", "", "", "Version of installer docker image to use while building locally")
+	cmd.Flags().StringVarP(&o.LocalInstallerDir, "installer-dir", "", "", "Directory of installer docker image to use while building locally")
```

Comment:
It is not the directory to the image but to the Dockerfile, isn't it?

Reply:
No, it's not path do Dockerfile, it's path to image. For example `eu.gcr.io/kyma-project/develop`, but you could provide your own if you run from PR or your dockerhub. Those two flags map 1-1 to ARGs from [kyma-installer Dockerfile](https://github.com/kyma-project/kyma/blob/master/tools/kyma-installer/kyma.Dockerfile)

Reply:
ok fine. Maybe path is then misleading and it is more a docker repository.. However, I'm fine :)